modules/storage-api/src/test/java/org/apache/ignite/internal/storage/TestMvTableStorageTest.java

```diff
@@ -32,7 +32,7 @@
 public class TestMvTableStorageTest extends AbstractMvTableStorageTest {
     @BeforeEach
     void setUp(
-            @InjectConfiguration("mock.tables.foo.dataStorage.name = " + TestStorageEngine.ENGINE_NAME)
+            @InjectConfiguration("mock.tables.foo{ partitions = 512, dataStorage.name = " + TestStorageEngine.ENGINE_NAME + "}")
```

Comment:
Why 512?

Reply:
All heirs of `org.apache.ignite.internal.storage.AbstractMvTableStorageTest` set this value, they missed it here, so some tests may fail.